app/components/MarketplaceForm.tsx

```diff
@@ -22,6 +22,8 @@ export function MarketplaceForm({ projects, tokens }: { projects: Project[]; tok
         </p>
       </section>

```

Comment:
Should this go in the features folder? Asking bc there's another form in there

Reply:
So here's what i was thinking:
`/components` is for anything UI kit-y. Buttons, inputs, cards, tooltips, etc. Essentially the "primitives" of our UI.

`/features` have higher levels of integration. Potential features are "MarketplaceForm", "LaborMarketSearcher", "TokenSelector". These features build on top of the components for a more specific usecase.

So to answer @termaatFlipside yes this probably should go in `features/marketplace-form/...`

Also see gumby's feature folder https://github.com/FlipsideCrypto/product-eng/tree/main/apps/gumby/app/features